picard/script/functions.py

```diff
@@ -1354,3 +1355,16 @@ def func_unique(parser, multi, case_sensitive="", separator=MULTI_VALUED_JOINER)
     if not case_sensitive:
         multi_value._multi = list({v.lower(): v for v in multi_value}.values())
     return multi_value.separator.join(sorted(set(multi_value)))
+
+
+@script_function(documentation=N_(
+    """`$countryname(country_code)`
+
+Returns the name of the country for the specified country code.  If the country code is invalid an empty string will be returned.
```

Comment:
IMHO it's important to mention here that the return value is always in english, independent of the language of Picard's UI.

Reply:
Actually, I'm thinking to wrap the output in `gettext_countries()` to return the translated value (similar to what is done in `picard.ui.options.releases.ReleasesOptionsPage`), but this breaks the tests because it isn't added to the `builtins` dictionary when the tests are executed.  Thoughts?

Reply:
Not entirely sure. I'd probably would not want the tag values to always change based on the UI language of Picard. A translate boolean flag as suggested by Sophist might be good.

Reply:
I was able to get the tests to work by importing the `picard.i18n` module.

Reply:
As suggested, I added an option parameter to determine whether or not the output is translated.